algrtm/src/components/OfferSection.js

```diff
@@ -0,0 +1,118 @@
+"use client";
+
+import React, { useState } from "react";
+import Icon from "./shared/Icon";
+import offerSectionTexts from "@/data/offerSectionsTexts";
+
+const OfferSection = () => {
+  const [sections, setSections] = useState(offerSectionTexts);
+
+  const handleOffer = (index) => {
+    setSections((prevSections) =>
+      prevSections.map((section, i) => ({
+        ...section,
+        visible: index === i ? !section.visible : false,
+      }))
+    );
+  };
+
+  return (
+    <section className="flex flex-col w-full mt-[8.5rem] gap-[2.75rem] justify-center items-center">
+      <h2 className="flex h2 text-neutralLight-neutral10 leading-[4rem]">
+        What we offer?
+      </h2>
+      <div className="flex flex-col gap-4 max-w-[78.25rem]">
```

Comment:
In Figma, this div is 94px away from the left and right edges of the screen. By using `max-w-[78.25rem]` we will not achieve the same size as in Figma and it will not be responsive. A better solution is to use `w-full px-[5.875rem]` so that you will always have the same padding on the left and right sides